docs/docs/feature-reference/output-streaming.md

```diff
@@ -0,0 +1,56 @@
+# Live-Streaming Tool Output
+
+For **synchronous** runs with select tools, Toolchest supports streaming tool output live to your terminal.
+
+By setting the argument `streaming_enabled=True`, lines will be printed to your terminal as they would otherwise be 
+printed by the tool to stdout and stderr.
```

Comment:
> "By setting the argument `streaming_enabled=True`, lines will be printed to your terminal as they would otherwise be 
printed by the tool to stdout and stderr."

This needs to be more concise. Maybe something like: "with streaming enabled, remote output is streamed live to where you're running Toolchest"